### PR TITLE
[AUT-4021] add missing helm parameters

### PR DESCRIPTION
**Pull request by ppodbielski-cloudentity**
## Description

Add missing helm parameters to values.yaml and update descriptions

## Type of changes

[] Bugfix (non-breaking change which fixes an issue)
[] New feature (non-breaking change which adds functionality)
[] Tests (extending the test suite)
[X] Refactor (internal improvement that doesn't change product functionality)
[] Other (if none of the other choices apply)

### CODE REVIEW

File: charts/acp/values.yaml [hunk 1]
```diff
@@ -81,108 +92,265 @@ ingress:
...
+  ##
   data: {}
   #  system:
   #     secret: mysecret
 
+## Dummy cert and key
+##
 certificate:
-  ## Dummy cert and key
+  ## Certyficate
```
Comment (jkotiuk) [inline charts/acp/values.yaml:353]: Certificate

File: charts/acp/values.yaml [hunk 2]
```diff
@@ -81,108 +92,265 @@ ingress:
...
+# tolerations: {}
+
 ## A security context defines privilege and access control settings for a Pod or Container
 ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+##
 securityContext:
   runAsUser: 0
 
-# ACP feature flags
+## ACP feature flags
```
Comment (mbilski) [inline charts/acp/values.yaml:163]: eventually, we should drop the features from values.yaml completely and redirect users to reference.yaml as we discussed

File: charts/acp/values.yaml [hunk 3]
```diff
@@ -81,108 +92,265 @@ ingress:
...
+  ##
   data: {}
   #  system:
   #     secret: mysecret
 
+## Dummy cert and key
+##
 certificate:
-  ## Dummy cert and key
+  ## Certificate:
```
Comment (mbilski) [inline charts/acp/values.yaml:353]: In the future, I would remove this example cert and key from values.yaml. 
We can keep them as default values in the templates.

**Review by mbilski — APPROVED** [review]

File: charts/acp/values.yaml (hunk 3 above)
Comment (ppodbielski-cloudentity) [reply charts/acp/values.yaml:353]: https://cloudentity.atlassian.net/browse/DEVOPS-1586

File: charts/acp/values.yaml (hunk 2 above)
Comment (ppodbielski-cloudentity) [reply charts/acp/values.yaml:163]: https://cloudentity.atlassian.net/browse/DEVOPS-1585

**Review by artursmolarek — APPROVED** [review]

**Review by jkotiuk — APPROVED** [review]

**Review by lisocki — APPROVED** [review]

**Review by wobara-cloudentity — APPROVED** [review]

**Pull request merged**